source/includes/administration/_notification_categories.md

```diff
@@ -0,0 +1,241 @@
+            }
+        ],
+        "icon": "fa fa-envelope",
+        "id": "dc921a2a-16c2-4bc3-8ed6-bcb571bae241",
+        "version": 1,
+        "updatedAt": "2020-10-29T14:06:50.000Z"
+    }
+}
+```
+Get a specific notification category.
+
+Optional Query Parameters | &nbsp;
+---------- | -----------
+`organization_id`<br/>*UUID* | The id of the organization we wish to display notification categories for. When left blank, the current user's notification categories will be shown.
+`language`<br/>*UUID* | Language of the notification categories. Values expected are of the form : "en" or "fr" as opposed to "english" or "french".
+
+Attributes | &nbsp;
+---------- | -----------
+`id`<br/>*UUID* | The id of the notification category.
+`organizationId`<br/>*UUID* | The organization id of the notification category.
+`createdAt`<br/>*string* | The date on which the notification category was created.
+`updatedAt`<br/>*string* | The date on which the notification category was updated.
+`icon`<br/>*string* | The icon used when displaying the notification category.
+`version`<br/>*integer* | The notification category's version.
+`translations`<br/>*Array[Object]* | The translations of the notification category's content.
+`translations.language`<br/>*string* | The language of the translation.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating (must be one of: title, body, description, url slug).
+`translations.id`<br/>*UUID* | The translation's id.
+`translations.version`<br/>*integer* | The translation's version.
+
+<!-------------------- CREATE NOTIFICATION CATEGORY -------------------->
+
+#### Create notification category
+
+`POST /content/categories/`
+
+```shell
+# Create notification category
+curl -X POST "https://cloudmc_endpoint/v2/rest/content/categories/" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json" \
+   -d "request-body"
+```
+> Request body example:
+
+```json
+{
+    "organizationId":"2cc1b5ba-2cfb-4bf9-9cb9-c67d34ba27b1",
+    "icon": "fa fa-envelope",
+    "translations": [
+        {
+            "language":"en",
+            "type":"title",
+            "text":"trial-ttt"
+        },
+        {
+            "language":"fr",
+            "type":"title",
+            "text":"trial-tt"
+        }
+    ]
+}
+```
+Create a notification category.
+
+Required | &nbsp;
+---------- | -----------
+`organizationId`<br/>*UUID* | The organization id of the notification category.
+`icon`<br/>*string* | The icon used when displaying the notification category.
+`translations`<br/>*Array[Object]* | The translations of the notification category's content.
+`translations.language`<br/>*string* | The language of the translation.
+`translations.text`<br/>*string* | The content of the notification.
+`translations.type`<br/>*string* | The content that we are translating (must be one of: title, body, description, url slug).
```

Comment:
Should only be title here

Reply:
fixed 65d3185551293faa443a5e59617d81963fb934f1